assets/alertmanager/alertmanager.yaml

```diff
@@ -94,6 +94,10 @@ spec:
         cpu: 1m
         memory: 20Mi
     terminationMessagePolicy: FallbackToLogsOnError
+  - name: config-reloader
```

Comment:
Is this supposed to be config-reloder sidecar? As the image says alertmanager?

Reply:
yes, alertmanager deployes `config-reloader`:
```
alertmanager-main-0
Container Name: alertmanager
resources: map[requests:map[cpu:4m memory:200Mi]]
Container Name: config-reloader
resources: map[requests:map[cpu:1m]]
Container Name: alertmanager-proxy
resources: map[requests:map[cpu:1m memory:20Mi]]
Container Name: kube-rbac-proxy
resources: map[requests:map[cpu:1m memory:20Mi]]
Container Name: prom-label-proxy
resources: map[requests:map[cpu:1m memory:20Mi]]
```